debian/libsaimetadata-dev.install

```diff
@@ -1,3 +1,5 @@
 meta/sai*.h usr/include/sai
 meta/Sai*.h usr/include/sai
 SAI/meta/sai*.h usr/include/sai
+usr/lib/*/libsaimetadata.so
+usr/lib/*/libsaimeta.so
```

Comment:
should those be in non-dev package?

Reply:
No, these files are only needed at linking time, not during runtime. At linking time, the SONAME of these libraries (which, I believe, are `libsaimetadata.so.0.0.0` and `libsaimeta.so.0.0.0` for these two libraries) gets recorded in the binary/library, and that file instead gets loaded.